File: PropertyManagementSystem/PropertyManagementSystem/Models/Community.cs
using System.ComponentModel.DataAnnotations;

namespace PropertyManagementSystem.Models
{
    public class Community: BaseModel
    {
        [Required, StringLength(50)]
        public string Name { get; set; }
        [Required]
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }

        [Required, StringLength(50)]
        public string City { get; set; }

        [Required, StringLength(50)]
        public string State { get; set; }

        [Required, StringLength(60)]
        public string Country { get; set; }

        [Required,StringLength(10)]
        public string Zipcode { get; set; }
    }
}

File: PropertyManagementUI/src/app/Models/Community.ts
import { BaseModel } from "./BaseModel";

export interface Community extends BaseModel {
    Name: string | null;
    AddressLine1: string | null;
    AddressLine2: string | null;
    City: string | null;
    State: string | null;
    Country: string | null;
    Zipcode: string | null;
}